docs/docs/guides/nuxt.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Nuxt.js + Hanko
+sidebar_label: Nuxt.js
+keywords: [nuxt, nuxtjs]
+sidebar_custom_props:
+  docCardIconName: nuxt
+---
+
+# Nuxt 3
+
+In this guide you will learn how to install and use the nuxt-hanko module by [Daniel Roe](https://github.com/danielroe/nuxt-hanko).
+The NuxtJS module provides a simple way to add Hanko elements and authentication to your NuxtJS application.
+
+:::info
+This guide and the nuxt-hanko module are only compatible with Nuxt 3. Find out more about Nuxt 3 at [Nuxt.com](https://nuxt.com)
+:::
+
+## Install dependencies
+
+Install the `nuxt-hanko` package:
+
+```shell npm2yarn
+npm install -D nuxt-hanko
+```
+
+## Configure NuxtJS module
+
+Add the `nuxt-hanko` module to the `modules` section of your `nuxt.config.ts` file.
+
+```jsx title="nuxt.config.ts" showLineNumbers
+export default defineNuxtConfig({
+  modules: ['nuxt-hanko'],
+  hanko: {
+    // You can also configure this by setting NUXT_PUBLIC_HANKO_API_URL environment variable at runtime
+    apiURL: '<YOUR_API_URL>'
+  }
+});
+```
+
+The module comes pre-configured with the options for most common use cases, you can override them by adding a `hanko` section to your `nuxt.config.ts` file. For all overrides please check the [module repo](https://github.com/danielroe/nuxt-hanko)
+
+:::info
+
+If you are using [Hanko Cloud](https://cloud.hanko.io), you can find the API URL on your project dashboard.
+If you are self-hosting you need to provide the URL of your running Hanko backend.
+
+:::
+
+## Hanko Components
+
+You can now use the Hanko components anywhere in your app: `<hanko-auth, <hanko-events> and <hanko-profile>`. These are web components that will be rendered on the client-side only. All the props are typed.
+
+You can turn auto-registration of components off (if you wish to use Hanko just on the server side or programmatically) by setting `hanko:{ registerComponents: false}` in your `nuxt.config.ts` file.
+
+### `<hanko-auth>` component
+
+To provide a login interface in your app, use the `<hanko-auth>` web component. This will render a login form and handle passkeys auth.
+
+```jsx title="login.vue" showLineNumbers
+<template>
+  <hanko-auth />
+</template>
+```
+
+By default no action is taken after a successful login. When you add the `hanko-logged-out` middleware to the login page it will automatically redirect the user to the page they were on before logging in, or to the optional `redirects.success` value in the module config.
```

Comment:
If you do not add the middleware to the page, no redirects will occur and will need to be configured manually with the `hanko-event` component

Adding the middleware will listen for hanko-events and redirect to `redirects.success` when login success event is triggered.

Reply:
Hmmm... testing now

Reply:
Tested, working as configured
Upon login; Redirects user to /user

If you are already logged in, and nav to /login it will redirect to path for `redirects.home`

@danielroe does it make more sense to redirect already logged in users to the success path instead?
`redirects.home` will be good for logout.

Reply:
Do you get the login page and manually trigger the form?
Or does it redirect you from login to / as soon as you nav to it?

Reply:
Ok... I can replicate

PR with fix is up: https://github.com/danielroe/nuxt-hanko/pull/9

Reply:
merged and released in 0.2.1 ✅